components/calendar/overview.md

```diff
@@ -92,11 +88,12 @@ The Blazor Calendar provides various parameters that allow you to configure the
 | `RangeEnd` | `DateTime` | The selected end date when [range selection]({%slug components/calendar/selection%}#range-selection-mode) is enabled. Supports two-way binding. |
 | `SelectedDates` | `List<DateTime>` | The selected dates when [multiple selection]({%slug components/calendar/selection%}#multiple-selection-mode) is used. |
 | `SelectionMode` | `CalendarSelectionMode` enum <br /> (`Single`) | The [selection mode]({%slug components/calendar/selection%}) of the calendar. |
+| `ShowWeekNumbers` | `bool` | Sets if the Calendar will display week numbers according to the [ISO-8601 format](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.isoweek.getweekofyear). Note that the [ISO week number may differ from the conventional .NET week number](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.calendar.getweekofyear). |
+| `ShowOtherMonthDays` | `bool` <br /> (`true`) | Defines whether the leading and trailing days from other months are visible within the current month. |
```

Comment:
The other month's days are not actually displayed IN / WITHIN the current month.

```suggestion
| `ShowOtherMonthDays` | `bool` <br /> (`true`) | Defines whether the leading and trailing days from other months are visible in the current month view. |
```